com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/parser/UriParametersParser.java

```diff
@@ -133,6 +133,8 @@ private void addHostBasedUris(Set<String> uris, String domain, List<String> host
             .stream()
             .map(routesMap -> (String) routesMap.get(SupportedParameters.ROUTE))
             .filter(Objects::nonNull)
+            .map(ApplicationURI::new)
+            .map(ApplicationURI::toString)
```

Comment:
Also, this can just be moved to line 145 as an else statement - if we don't want to modify the route - we still strip the scheme at least (the modify route logic will do it regardless)